pandas/tests/extension/test_integer.py

```diff
@@ -168,6 +168,15 @@ def check_opname(self, s, op_name, other, exc=None):
     def _compare_other(self, s, data, op_name, other):
         self.check_opname(s, op_name, other)
 
+    def test_can_compare_to_string(self):
+        # GH 28930
+        s = pd.Series([0, None], dtype="Int64")
+
+        result = s == "a"
+        expected = pd.Series([False, False])
```

Comment:
can you check that the other comparisons work correctly

Reply:
I think I found other bugs when adding more test cases, so I created a couple follow-up issues: https://github.com/pandas-dev/pandas/issues/28980, https://github.com/pandas-dev/pandas/issues/28981.

Reply:
Can you still use `all_compare_operators` here and just xfail where appropriate? Will be easier to track over time instead of piecing in various tests as fixes come along